src/views/TabView/TabView.js

```diff
@@ -20,7 +20,6 @@ export type TabViewConfig = {
   tabBarComponent?: ReactClass<*>,
   tabBarPosition?: 'top' | 'bottom',
   tabBarOptions?: {},
-  initialLayout?: { width: number, height: number },
```

Comment:
keep this here, but change the type to `Layout` which can be imported from `import type { Layout } from 'react-native-tab-view/src/TabViewTypeDefinitions';`

Reply:
It's already in the props type 3 lines below ;)

Reply:
huh, good catch :)